src/components/HistoryEventDetails/layouts/WalletEventDetails.js

```diff
@@ -103,7 +97,7 @@ function WalletEventDetails({ event }: Props) {
         <Text variant="large">{t('label.created')}</Text>
         <Spacing h={spacing.extraLarge} />
 
-        {isActivated ? (
+        {isDeployed ? (
```

Comment:
I'm not sure it should be checked for Etherspot in this place? The "activation" is only meant for Archanova and for Etherspot we will have deployment intro screen with no actual deployment (per our last call with Kieran). So until we don't have that implemented I think it's best not to let users enter that screen from Etherspot since it's meant for Archanova deploymeny only.

Reply:
Removed that, as new history UI is only for Etherspot.